src/panels.cpp

```diff
@@ -1166,7 +1166,7 @@ static void draw_char( avatar &u, const catacurses::window &w )
     mvwprintz( w, 2, 1, c_light_gray, _( "Focus:" ) );
     mvwprintz( w, 0, 19, c_light_gray, _( "Mood :" ) );
     mvwprintz( w, 1, 19, c_light_gray, _( "Speed:" ) );
-    mvwprintz( w, 2, 19, c_light_gray, _( "Move :" ) );
+    mvwprintz( w, 2, 19, (u.get_movement_mode() == "run" ? c_red : (u.get_movement_mode() == "crouch" ? c_dark_gray : c_light_gray)), _( "Move :" ) );
```

Comment:
This would be a little clearer if you captured the color on a seperate line, like
nc_color move_mode_color = u.get_movement_etc...
and then passed that variable to both print statements.

Reply:
If there are still things that aren't working tomorrow I will rewrite it like that, but for now I just want to make sure I understand the checking process.